mmcv/parallel/scatter_gather.py

```diff
@@ -24,7 +24,14 @@ def scatter_map(obj):
             if obj.cpu_only:
                 return obj.data
             else:
-                return Scatter.forward(target_gpus, obj.data)
+                data = obj.data
+                if isinstance(data, torch.Tensor):
```

Comment:
Should update the Scatter rather than this function.

Reply:
`list(map(list, zip(*map(scatter_map, data))` It will call internal functions in a loop.